### PR TITLE
Remove column names from generated csv voter files

### PR DESCRIPTION
**Pull request by cubewhiz**
This seemed simple enough so I gave it a shot. Let me know if I did this wrong or broke something.

From Walker via email:
I just want to make a minor change to the voting files that are generated when I click on each one.

In each file, it has the first row as a list of: Name, email, ID. Would I be able to get that row removed? 

When I try to upload the file to Helios, it tries to add that row into the data as a voter. So each time I have to take it out of the data before uploading the file. Sometimes messing around with the file also makes it so the file doesn't upload correctly in Helios.

### CODE REVIEW

**Comment by jfly** [comment]
@cubewhiz thanks for giving this a shot! I believe all you've done here is update our test (aka "spec"). You'll also need to update the relevant code that *generates* the csv. That codes over in https://github.com/thewca/worldcubeassociation.org/blob/91327af231729416552f36a216617469eb0d673d/WcaOnRails/app/controllers/admin_controller.rb#L99.

**Comment by cubewhiz** [comment]
Oops, is that better?

File: WcaOnRails/app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -98,7 +98,6 @@ def leader_senior_voters
 
   private def voters(users, filename)
     csv = CSV.generate do |line|
-      line << ["id", "email", "name"]
```
Comment (AlbertoPdRF) [inline WcaOnRails/app/controllers/admin_controller.rb]: If I get it right, Travis isn't passing because it doesn't like this empty line here.

**Comment by jfly** [comment]
Thanks, Bob! This looks good to me.

**Pull request merged**